internal/pkg/expanders/containerRegistry.go

```diff
@@ -440,7 +506,7 @@ func (e *ContainerRegistryExpander) getItemsForURL(ctx context.Context, url stri
 	span, ctx := tracing.StartSpanFromContext(ctx, "getItemsForURL(containerregistry):"+url, tracing.SetTag("url", url))
 	defer span.Finish()
 
-	responseBuf, err := e.doRequest(ctx, url, accessToken)
+	responseBuf, err := e.doRequest(ctx, "GET", url, accessToken)
```

Comment:
nit: move these verbs to consts

Reply:
I'm not sure how much it adds moving these to consts - I find them quite easy to parse (and locate) as strings. I also had a quick look at the net/http code and that just uses plain strings :-)